deployment-template/ecs/aws-otel-ec2-sidecar-deployment-cfn.yaml

```diff
@@ -58,52 +66,49 @@ Resources:
           - !Ref ExecutionRoleArn
       NetworkMode: awsvpc
```

Comment:
I think `awsvpc`, `bridge`, `host` - all are recommended network modes fo `ECS`, so any one of them would be ok - as long as the template is complete and works. This template was already in `awsvpc` network mode, but in my opinion this should be fine as Fargate only allows `awsvpc` network mode and so this would allow 0 change for network access between services when moving from `EC2` to `Fargate`. Also I think, `ENI's` offered by `awsvpc` for each container is more useful/powerful than `bridge links`.  Also all other `ECS-EC2` setup instructions (like JSON and other templates) are also already published for `awsvpc` network mode only.

Reply:
So that means that aws-otel-ec2-sidecar-deployment-cfn.yaml is broken and aws-otel-fargate-sidecar-deployment-cfn.yaml is not?

@bryan-aguilar I think there is value in demonstrating how it works with EC2 and Fargate. Having said that, would it work if we make the type of Launch Type configurable?

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-conditions.html

@PaurushGarg what do you think? The benefit is that we would be able to remove lots of duplicated code.

Reply:
Please disregard my comment as this might be outside of the scope of this pr.

Reply:
Thanks. Updated the template to use bridge mode instead.